### PR TITLE
3.4: Modularize Configuring Hosts using Ansible

### PR DESCRIPTION
**Pull request by maximiliankolb**
Cherry-picks from https://github.com/theforeman/foreman-documentation/pull/2078

Only the first commit had conflicts which I've (hopefully correctly) resolved:

commit 71137cbadd18aa32352861512b45387e717773b5
conflicts on 3.4:

	both modified:   guides/common/assembly_configuring-and-setting-up-remote-jobs.adoc
	both modified:   guides/common/assembly_getting-started-with-ansible.adoc
	both modified:   guides/common/modules/proc_configuring-your-project-to-run-ansible-roles.adoc
	deleted by us:   guides/common/modules/proc_setting-the-job-rate-limit-on-smartproxy.adoc
	deleted by us:   guides/common/modules/ref_customizing-ansible-configuration.adoc

### CODE REVIEW

File: guides/common/modules/proc_monitoring-remote-jobs.adoc [hunk 1]
```diff
@@ -18,30 +17,26 @@ This displays the list of hosts on which the job is running.
...
 ----
-
 . Monitor the job output:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # hammer job-invocation output \
---id _job_ID_ \
---host _host_name_
+--host _My_Host_Name_ \
```
Comment (adamlazik1) [inline guides/common/modules/proc_monitoring-remote-jobs.adoc:33]:
```suggestion
--host "_My_Host_Name_" \
```

**Review by adamlazik1 — APPROVED** [review]
Thanks, LGTM! Only two small notes.

**Comment by maximiliankolb** [comment]
Rebased to master and applied one suggestion. I can implement the second idea in a follow-up PR.

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked to 3.3:
   03ddb9011..fec46e69a  3.3 -> 3.3

**Pull request merged**